cluster/images/canton-sequencer/app.conf

```diff
@@ -78,6 +78,8 @@ canton {
             # enable https://github.com/DACH-NY/canton/pull/27661
             buffer-payloads = true
             buffer-events-with-payloads = false
+            # higher size than the max of a cometbft block
```

Comment:
That image is used for the non-cometbft sequencer so you might just wanna omit it or adjust the comment?